app/models/internal/reports/Recipient.java

```diff
@@ -25,6 +26,7 @@
  *
  * @author Christian Briones (cbriones at dropbox dot com)
  */
+@Loggable
```

Comment:
Typically, we put the annotation on the implementation because it will serialize the implementation instance not the interface. I don't think this is wrong, in fact it's probably more right because it _should_ constrain the log output to just the methods on the interface. 

If it's not too much trouble, could you post output of logging an implementation instance of `Recipient` where the implementation has a public getter on it (e.g. `public String getFoo() { return "foo"; }`)?

Reply:
Rather than opening this possibly-time-consuming can of worms, I just put the annotation on `DefaultRecipient`. I can open an issue for this if you want?